### ISSUE TITLE
Rustler crashes out on the loading screen after you start a new game

### ISSUE THREAD

**Issue by benderscruffy** [yuzu-emu/yuzu#11758]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Mainline 1585

### Description of Issue

start a new game and it crashes out on the loading screen

### Expected Behavior

goes ingame

### Reproduction Steps

run the game

### Log File

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/12879030/yuzu_log.txt)

### System Configuration

AMD Ryzen 7 5700X 8-Core Processor | 16 Threads | 31.92 GiB RAM | TSC: 3.400GHz | AVX+ | FMA3
Operating system: Windows, Major: 10, Minor: 0, Build: 22621, Service Pack: none, Compatibility mode: 0
Vulkan-compatible GPU: 'NVIDIA GeForce RTX 3060' running on driver 537.58.0.0

**Comment by Squall-Leonhart** [comment]
devs might need a crash dump for this one.

**Comment by german77** [comment]
Not really. Yuzu didn't crash. Emulation crashed which already provides a call stack.  This game tried to access `google-analytics`. Which could be likely the motive of crash. Requires a deeper look to make sure.
